testsuite/MDAnalysisTests/topology/test_fetch_pdb.py

```diff
@@ -0,0 +1,84 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the Lesser GNU Public Licence, v2.1 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+# Note need to make test not run when there is no internet
+
+from requests.exceptions import HTTPError
+from urllib import request
+
+import MDAnalysis as mda
+import pytest
+
+
+
+def has_pooch():
+    try:
+        import pooch
+        return True
+    except ModuleNotFoundError:
+        return False
+    
+
+def check_internet():
+    try:
+        request.urlopen('https://files.wwpdb.org/', timeout=2)
+        return True
+    except request.URLError as err: 
+        return False
+    
+    
+@pytest.mark.skipif(has_pooch() is False or check_internet() is False, reason="Cannot connect to https://files.wwpdb.org/'")
+class TestDocstringExamples:
+    """This class test the example found in fetch_pdb's docstring"""
+
+    def test_one_file_download(self, tmp_path):
+        """Tests docstring's mda.fetch_pdb("1AKE", file_format="cif") """
```

Comment:
Rewrite the doc strings ... they sound generated and are just adding noise. You can probably omit them